contracts/StateRelayer.sol

```diff
@@ -78,17 +79,6 @@ contract StateRelayer is UUPSUpgradeable, AccessControlUpgradeable {
     }
     MasterNodeInformation private masterNodeInformation;
 
-    // Events
-    event UpdateDEXInfo(
-        string[] dex,
```

Comment:
We should keep these events, or atleast emit an event with fewer data so that it is possible to get a list of timestamps/block numbers at which the data was updated.